.github/workflows/xlsx-to-csv.yml

```
@@ -14,46 +14,81 @@ on:
-          echo "Converted $xlsx_file to $csv_file"
-        done
-
-    - name: Commit and push changes
-      run: |
-        cd mosip-data
-        git branch
-        git add .
-        git commit -s -m "Added converted CSV files" || echo "No changes to commit"
-        git push "https://x-access-token:${{ secrets.ACTION_PAT }}@github.com/${{ github.event.pull_request.head.repo.full_name }}.git" HEAD:${{ github.event.pull_request.head.ref }}
+      - name: Clone the repository
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)